docs/scripting-ref.rst

```diff
@@ -917,6 +917,84 @@ If your script returns the result of ``splash:png()`` in a top-level
 ``"png"`` key (as we've done in a previous example) then Splash UI
 will display it as an image.
 
+.. _splash-jpeg:
+
+splash:jpeg
+-----------
+
+Return a `width x height` screenshot of a current page in JPEG format.
+
+**Signature:** ``jpeg = splash:jpeg{width=nil, height=nil, render_all=false, scale_method='raster', quality=75}``
+
+**Parameters:**
+
+* width - optional, width of a screenshot in pixels;
+* height - optional, height of a screenshot in pixels;
+* render_all - optional, if ``true`` render the whole webpage;
+* scale_method - optional, method to use when resizing the image, ``'raster'``
+  or ``'vector'``
+* quality - optional, quality of JPEG image, integer in range from ``0`` to ``100``
+
+**Returns:** JPEG screenshot data.
+
+**Async:** no.
+
+Without arguments ``splash:jpeg()`` will take a snapshot of the current viewport.
```

Comment:
There's one more trick we can do with docs for these 2 methods - we can create some section related to image processing in docs with common howtos and limitations, remove some verbose parts from parameters docs in script reference docs - leave only concise one line description - and reference newly created section where it applies.

+1 to moving this problem out of the current PR's scope